src/main/kotlin/eu/europa/ec/eudi/openid4vci/AuthorizeIssuance.kt

```diff
@@ -113,6 +116,13 @@ sealed interface AuthorizedRequest : java.io.Serializable {
     ) : AuthorizedRequest
 }
 
+sealed interface AuthorizationDetailsInTokenRequest {
```

Comment:
I would call it something like AccessTokenOption, AsRequested  for DoNotInclude, Limited for Include

Reply:
Done